README.md

```diff
@@ -1,33 +1,27 @@
-# Change file mode to executable
-chmod +x ./scripts/bootstrap.sh
-# Download binaries and docker images
-./scripts/bootstrap.sh [version] [ca version] [thirdparty_version]
-```
+# Hyperledger Fabric Samples
+
+You can use Fabric samples to get started working with Hyperledger Fabric, explore important Fabric features, and learn how to build applications that can interact blockchain networks using the Fabric SDK's. To learn more about Hyperledger Fabric, visit the [Fabric documentation](https://hyperledger-fabric.readthedocs.io/en/master).
+
+## Getting started with the Fabric samples
+
+To use the Fabric samples, you need to download the Fabric Docker images and the Fabric CLI tools. First, make sure that you have installed all of the [Fabric prerequisites](https://hyperledger-fabric.readthedocs.io/en/master/prereqs.html). You can then follow the instructions to [Install the Fabric Samples, Binaries, and Docker Images](https://hyperledger-fabric.readthedocs.io/en/master/install.html) in the Fabric documentation. In addition to downloading the Fabric images and tool binaries, the instructions will clone the Fabric samples on your local machine.
+
+## Guide to the Fabric samples
+
+You can use the following table to learn more about each sample, and find the corresponding tutorial or documentation.
+
+|  **Sample** | **Description** | **Documentation** |
+| -------------|------------------------------|------------------|
+| [Fabric test network](test-network) | Get started by deploying a basic Fabric network on your local machine. | [Using the Fabric test network](https://hyperledger-fabric.readthedocs.io/en/master/test_network.html) |
+| [Fabcar](fabcar) | Lean how to use the Fabric SDK's to invoke smart contracts from your client applications. | [Writing your first application](https://hyperledger-fabric.readthedocs.io/en/master/write_first_app.html) |
+| [Commercial paper](commercial-paper) | Explore a use case in which two organizations use a blockchain network to trade commercial paper. | [Commercial paper tutorial](https://hyperledger-fabric.readthedocs.io/en/master/tutorial/commercial_paper.html) |
+| [Interest rate swaps](interest_rate_swaps) | Explore state based endorsement using a financial services use case. | [Setting Key level endorsement policies](https://hyperledger-fabric.readthedocs.io/en/master/endorsement-policies.html#setting-key-level-endorsement-policies) |
+| [Off chain data](off_chain_data) | Learn how to use the Peer channel-based event services to build an off chain database for reporting and analytics. | [Peer channel-based event services](https://hyperledger-fabric.readthedocs.io/en/master/peer_event_services.html) |
+| [High throughput](high-throughput) | Learn how you can design your smart contracts to process a large volume of transactions. | |
+| [First network](first-network) | **Deprecated** | [Build your first network](https://hyperledger-fabric.readthedocs.io/en/master/build_network.html) |
```

Comment:
I think it's worth adding something like "Deprecated in favor of Fabric test network."